bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/MagnitudeEditor.java

```diff
@@ -243,11 +243,16 @@ public void setTarget(SPTarget target, Mode mode) {
             // passband combo box.  It is never editable but appears to become
             // editable after the user selects a passband and the MagAddRow
             // is replaced with a MagEditRow.  \u2192 is an right pointing arrow
-            tf = new JTextField("Select \u2192") {{
-                setColumns(5);
-                setEnabled(false);
-                setMinimumSize(getPreferredSize());
-            }};
+            tf = new JTextField("Select \u2192") {
+                {
+                    setColumns(5);
+                    setEnabled(false);
+                    setMinimumSize(getPreferredSize());
+                }
+                @Override public void setEnabled(boolean enabled) {
+                    super.setEnabled(false); // never enable
+                }
+            };
```

Comment:
Very nice.  I guess this text field is only used to suggest editing and show the "Select ->" line? That is, once the band is selected it gets swapped out for a normal text field.